services/QuillLMS/engines/evidence/lib/evidence/evidence/synthetic/seed_data_generator.rb

```diff
@@ -119,6 +119,7 @@ def run
           .uniq {|r| r.text }
           .reject {|r| r.text.in?(current_result_texts)}
           .reject {|r| regex_exclude?(r.text) }
+          .reject {|r| opinion_api_flagged?(r.text) }
```

Comment:
@happythenewsad This will hit the opinion API a few hundred times in rapid succession. Is there a concern with load here?